### ISSUE TITLE
Verify and resolve accessibility violations

### ISSUE THREAD

**Issue by hortongn** [uclibs/ucrate#419]
### Descriptive summary

On scholar_uc we resolved several accessibility violations that were found by a scan.  We should check to see which of those violations still exist in ucrate and apply the same fixes, if needed.

For each of the scholar_uc PRs below, 1) see if the violation exists in ucrate and 2) apply the fix

Resolve table header accessibility violations
https://github.com/uclibs/scholar_uc/pull/1601

Resolve form accessibility violations
https://github.com/uclibs/scholar_uc/pull/1599

contrast violations
https://github.com/uclibs/scholar_uc/pull/1593

Fix contrast issues missed by scan
https://github.com/uclibs/scholar_uc/pull/1606

Fix Meta Tag violations from accessibility scan
https://github.com/uclibs/scholar_uc/pull/1595

Resolve ARIA accessibility issues
https://github.com/uclibs/scholar_uc/pull/1605

**Comment by crowesn** [comment]
@hortongn 

Might we be able to run a fresh scan on dev branch of Ucrate?

**Comment by crowesn** [comment]
Waiting for information about accessibility scan.
